pkg/kyma/cmd/test/cmd.go

```diff
@@ -0,0 +1,155 @@
+package test
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/kyma-incubator/kymactl/internal"
+	kyma_helm "github.com/kyma-incubator/kymactl/internal/helm"
+	"github.com/kyma-incubator/kymactl/internal/step"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/helm/pkg/helm"
+	"k8s.io/helm/pkg/helm/environment"
+	helm_release "k8s.io/helm/pkg/proto/hapi/release"
+)
+
+type Options struct {
+	step.Factory
+	skip []string
+	verbose bool
+}
+
+var testNamespaces = []string{
+	"kyma-system",
+	"istio-system",
+	"knative-serving",
+	"kyma-integration",
+}
+
+var testReleases = map[string]bool{
+	"core": true,
+	"monitoring": false,
+	"logging": false,
+	"istio": true,
+	"knative": true,
+	"application-connector": true,
+}
+
+//NewCmd creates a new install command
+func NewCmd() *cobra.Command {
+	opts := &Options{}
+	cmd := &cobra.Command{
+		Use:   "test",
+		Short: "test kyma installation",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringArrayVar(&opts.skip, "skip", []string{}, "Skip tests for these releases")
+	cmd.Flags().BoolVar(&opts.NonInteractive, "non-interactive", false, "Do not use spinners")
```

Comment:
am thinking if the non-interactive mode for your test command has the same semantic like the silent mode for minikube install. I think there are different as I might want to disable the dialogues (do you really want to delete) but still want to have nice spinner. But that is maybe so rare that we still should unify them to a "non-interactive" but what could be a short key?

Reply:
In case of dialogues I would use something like `--default/-d` which would always use less intrusive action and `--force/-f` to do the opposite. Not sure why, but people tend to use `--force` more frequently then its counterpart which often is not even implemented :D

Reply:
For simplicity I used the non-interactive flag for now. Maybe the --force is better indeed.